testsuite/smoke-test/src/nft_transaction.rs

```diff
@@ -50,7 +50,7 @@ impl AptosTest for NFTTransaction {
                     creator.address(),
                     0,
                     0,
```

Comment:
should move this to a move e2e test

Reply:
let's keep the scope of this PR as it is and tackle the test migration in a separate PR.
1. we need to move the entire NFT smoke-test to move-e2e test, 2 we need to use sdk builder there as well.

Reply:
Yes, started a local node and send some dummy TXNs. I also started a local indexer tailing from my local node to test the NFT events parsing and see the data populated in my local postgres DB